src/main/java/com/gamegoo/service/manner/MannerService.java

```diff
@@ -252,9 +292,78 @@ public MannerRating update(MannerRequest.mannerUpdateDTO request, Long memberId,
                     mannerRatingKeyword.setMannerKeyword(mannerKeyword);
                 }
             }
+
+            // 매너점수 산정.
+            int mannerScore = updateMannerScore(targetMember);
+
+            // 매너레벨 결정.
+            int mannerLevel = mannerLevel(mannerScore);
+
+            // 매너레벨 반영.
+            targetMember.setMannerLevel(mannerLevel);
+
+            // db 저장.
+            memberRepository.save(targetMember);
+
             return mannerRatingRepository.save(mannerRating);
         }
+    }
+
+    // 매너점수를 산정하고 업데이트.
+    public int updateMannerScore(Member targetMember){
+
+        // 매너평가 ID 조회
+        List<MannerRating> mannerRatings = mannerRatingRepository.findByToMemberId(targetMember.getId());
+
+        int totalCount;
+
+        // 매너 평가 + 비매너 평가를 처음 받은 회원
+        if (mannerRatings.size()==1){
+            if (mannerRatings.get(0).getIsPositive()) {
+                totalCount = mannerRatings.get(0).getMannerRatingKeywordList().stream()
+                        .map(mannerRatingKeyword -> mannerRatingKeyword.getMannerKeyword().getId())
+                        .collect(Collectors.toList()).size();
+            } else {
+                totalCount = (mannerRatings.get(0).getMannerRatingKeywordList().stream()
+                        .map(mannerRatingKeyword -> mannerRatingKeyword.getMannerKeyword().getId())
+                        .collect(Collectors.toList()).size())*-2;
+            }
+        } else {
+            List<Long> positiveMannerKeywordIds = mannerRatings.stream()
+                    .filter(MannerRating::getIsPositive)
+                    .flatMap(mannerRating -> mannerRating.getMannerRatingKeywordList().stream())
+                    .map(mannerRatingKeyword -> mannerRatingKeyword.getMannerKeyword().getId())
+                    .collect(Collectors.toList());
```

Comment:
여기도 mannerKeyword의 id가 특별히 필요하지 않다면, 
```suggestion
            int positiveCount = mannerRatings.stream()
                .filter(MannerRating::getIsPositive)
                .flatMap(mannerRating -> mannerRating.getMannerRatingKeywordList().stream())
                .collect(Collectors.toList())
                .size();
```
이렇게 수정해도 될 것 같습니다!